### ISSUE TITLE
Typo and question on citing RDTools

### ISSUE THREAD

**Issue by matsuobasho** [NREL/rdtools#414]
**Describe the bug**
The citing RDtools section in [Readthedocs](https://rdtools.readthedocs.io/en/stable/) has  a typo:
```
Michael G. Deceglie, Ambarish Nag, Adam Shinn, Gregory Kimball, Daniel Ruth, Dirk Jordan, Jiyang Yan, Kevin Anderson, Kirsten Perry, Mark Mikofski, Matthew Muller, Will Vining, and Chris Deline RdTools, version {insert version}, Compuer Software, https://github.com/NREL/rdtools. DOI:{insert DOI}
```

Should be Computer Software

**Additional context**
Also, there is no period between Chris Deline and RDtools.  Additionally, what is the difference between the version number and the DOI?

**Comment by mdeceglie** [comment]
Thanks for catching those typos, we'll get that updated.

The DOI is a digital object identifier that you can get from the project's [zenodo page](https://zenodo.org/doi/10.5281/zenodo.1210316). It provides a static archival link to the files corresponding to a specific version. The version number is just from the RdTools versioning system. For example, to cite version 3.0.0-alpha.4 you would use:

Michael G. Deceglie, Ambarish Nag, Adam Shinn, Gregory Kimball, Daniel Ruth, Dirk Jordan, Jiyang Yan, Kevin Anderson, Kirsten Perry, Mark Mikofski, Matthew Muller, Will Vining, and Chris Deline. RdTools, Version 3.0.0-alpha.4, Computer Software, https://github.com/NREL/rdtools. DOI: 10.5281/zenodo.10247179

**Comment by martin-springer** [comment]
resolved with #421
